python/tvm/relay/backend/executor_factory.py

```diff
@@ -75,6 +75,8 @@ class AOTExecutorFactoryModule(ExecutorFactoryModule):
     ----------
     ir_mod : :py:class:`~tvm.IRModule`
         The IR module to build.
+    built_ir_mods : dict[Target, IRModule]
+        The IR modules built per Target.
```

Comment:
What does IR modules "built" mean here ?  When IRModule are built wont they become runtime.Modules ?

Reply:
`lowered` ?

Reply:
Changed to `lowered_ir_mods` :smile_cat: